functions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Links the PHP to the correct Database
+ *
+ * @param none
+ *
+ * @return PDO, returning link to database
+ */
+function getDatabaseObject(): PDO {
+
+    return $db = new PDO('mysql:host=db;dbname=pokedex', 'root', 'password');
```

Comment:
return means the rest of the function will not execute, so the set attribute may not work. you should return `$db` after the set attribute

Reply:
Amended